float64[2] rpm_to_voltage_gain
float64 p_gain
float64 i_gain
float64 d_gain
float64 min_voltage
float64 max_voltage